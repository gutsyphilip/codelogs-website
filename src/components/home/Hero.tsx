import AvatarImage from '../../assets/images/avatar.jpg'
import { Button } from '../utility/button'
import { FeaturedCard } from '../utility/cards'
import React from 'react'
import { cardVariantTypes } from '../utility/cards/card.type'
import styles from './home.module.scss'

const Hero = () => {
  return (
    <section className={styles.hero}>
      <div className={styles.hero_caption}>
        <h2 className={styles.hero_caption_title}>Welcome to Codelogs!</h2>
        <p className={styles.hero_caption_description}>
          Quality streams on advanced software engineering concepts taught by
          experienced professionals.
        </p>
        <Button className={styles.hero_caption_button} variant="themed">
          Watch videos
        </Button>
      </div>

      <div>
        <FeaturedCard
          className={styles.hero_upNext}
          data={{
            date: 'Saturday, May 2 2020 - 18:00 WAT',
            label: 'COMING SOON',
            title: 'Defining your Testing Strategy using the Testing Pyramid.',
            image: AvatarImage
          }}
          hasPadding
          variant={cardVariantTypes.light}
        />
      </div>
    </section>
  )
}

export default Hero
