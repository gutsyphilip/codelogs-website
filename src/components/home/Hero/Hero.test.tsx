import React from "react"
import Hero from "./Hero"
import renderer from "react-test-renderer"

describe("Hero", () => {
    it("should always render the same markup", () => {
        const tree = renderer.create(<Hero />).toJSON
        expect(tree).toMatchSnapshot()
    })
})
