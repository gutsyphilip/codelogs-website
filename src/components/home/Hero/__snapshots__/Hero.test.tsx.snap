// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero should always render the same markup 1`] = `[Function]`;
